ingestion/src/metadata/ingestion/source/mssql_usage.py

```diff
@@ -17,60 +17,36 @@
 from metadata.generated.schema.entity.services.connections.database.mssqlConnection import (
     MssqlConnection,
 )
-from metadata.generated.schema.entity.services.connections.metadata.openMetadataConnection import (
-    OpenMetadataConnection,
-)
 from metadata.generated.schema.entity.services.databaseService import (
     DatabaseServiceType,
 )
 from metadata.generated.schema.metadataIngestion.workflow import (
     Source as WorkflowSource,
 )
+from metadata.generated.schema.metadataIngestion.workflow import WorkflowConfig
 from metadata.ingestion.api.source import InvalidSourceException, Source, SourceStatus
 
 # This import verifies that the dependencies are available.
 from metadata.ingestion.models.table_queries import TableQuery
-from metadata.ingestion.source.sql_alchemy_helper import (
-    SQLAlchemyHelper,
-    SQLSourceStatus,
-)
+from metadata.ingestion.source.sql_alchemy_helper import SQLSourceStatus
+from metadata.utils.engines import get_engine
 from metadata.utils.helpers import get_raw_extract_iter, get_start_and_end
 from metadata.utils.sql_queries import MSSQL_SQL_USAGE_STATEMENT
 
 
 class MssqlUsageSource(Source[TableQuery]):
-    """
-    MSSQL Usage source
-
-    Args:
-        config:
-        metadata_config:
-
-    Attributes:
-        config:
-        analysis_date:
-        sql_stmt:
-        alchemy_helper:
-        report:
-    """
-
-    def __init__(self, config: WorkflowSource, metadata_config: OpenMetadataConnection):
+    def __init__(self, config: WorkflowSource, metadata_config: WorkflowConfig):
         super().__init__()
         self.config = config
         self.connection = config.serviceConnection.__root__.config
         start, end = get_start_and_end(self.config.sourceConfig.config.queryLogDuration)
         self.analysis_date = start
         self.sql_stmt = MSSQL_SQL_USAGE_STATEMENT.format(start_date=start, end_date=end)
-        self.alchemy_helper = SQLAlchemyHelper(
-            self.connection,
-            metadata_config,
-            DatabaseServiceType.Mssql.value,
-            self.sql_stmt,
-        )
         self.report = SQLSourceStatus()
+        self.engine_connecter = get_engine(self.config.serviceConnection)
```

Comment:
Can we keep the name just as engine? TO be consistent with other sources

Reply:
@pmbrull Sure